templates/home.html

```diff
@@ -95,12 +95,12 @@
                     <div class="table-body-cell">Адрес:</div>
                     <div class="table-body-cell">
                         {{ $ac.Name }}
-                        <button onclick="fetch('http://{{ $.HostIP }}:{{ $.Port }}/door?placeID={{ $element.Place.ID }}&accessControlID={{ $ac.ID }}'); return false;">Открыть</button>
+                        <button onclick="fetch('{{ $.HassioIngress }}/door?placeID={{ $element.Place.ID }}&accessControlID={{ $ac.ID }}'); return false;">Открыть</button>
```

Comment:
We can just remove HassioIngress because it's empty in docker container and (I guess) we can send requests relative to root URL in HA too

  ```suggestion
                        <button onclick="fetch('/door?placeID={{ $element.Place.ID }}&accessControlID={{ $ac.ID }}'); return false;">Открыть</button>
```

Reply:
X-Ingress-Path Header use in HA addons and this var must be for use suburl (info from HA forum, not have HA)